packages/orbit-components/src/Dialog/README.md

```diff
@@ -19,13 +19,14 @@ After adding import into your project you can use it simply like:
-| dataTest          | `string`                | Optional prop for testing purposes.                                                                    |
-| renderInPortal    | `boolean`               | Optional prop, set it to `false` if you're rendering Dialog inside a custom portal, defaults to `true` |
-| description       | `React.Node`            | Optional description of the main action that Dialog performs.                                          |
-| illustration      | `React.Node`            | Optional illustration of the Dialog.                                                                   |
-| **primaryAction** | `React.Node`            | Primary and required action that user can do with the Dialog.                                          |
-| secondaryAction   | `React.Node`            | Optional, secondary action that user can perform - possibility to close the Dialog most of the time.   |
-| onClose           | `() => void \| Promise` | The title of the Dialog - preferably the purpose of the main action.                                   |
-| **title**         | `React.Node`            | The title of the Dialog - preferably the purpose of the main action.                                   |
+| Name              | Type                    | Default | Description                                                                                                                         |
+| :---------------- | :---------------------- | :------ | :---------------------------------------------------------------------------------------------------------------------------------- |
+| dataTest          | `string`                |         | Optional prop for testing purposes.                                                                                                 |
+| renderInPortal    | `boolean`               |         | Optional prop, set it to `false` if you're rendering Dialog inside a custom portal, defaults to `true`                              |
+| description       | `React.Node`            |         | Optional description of the main action that Dialog performs.                                                                       |
+| illustration      | `React.Node`            |         | Optional illustration of the Dialog.                                                                                                |
+| **primaryAction** | `React.Node`            |         | Primary and required action that user can do with the Dialog.                                                                       |
+| secondaryAction   | `React.Node`            |         | Optional, secondary action that user can perform - possibility to close the Dialog most of the time.                                |
+| lockScrolling     | `boolean`               | `true`  | Whether to prevent scrolling of the rest of the page while Dialog is open. This is on by default to provide better user experience. |
```

Comment:
```suggestion
| lockScrolling     | `boolean`               | `true`  | Whether to prevent scrolling of the rest of the page while Dialog is open. This is on by default to provide a better user experience. |
```